.travis.yml

```diff
@@ -16,6 +16,7 @@ git:
 branches:
   only:
     - master
+    - fix-shadowing
```

Comment:
I've noticed that the branch also still exists, keep or remove?

Reply:
removed